test/ilpQuoterSpec.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+const nock = require('nock')
+nock.enableNetConnect(['localhost'])
```

Comment:
Ok, though this is not functional code, it's tests written on top of a DSL and I don't think I want to spend too much time understanding that stuff in details. It's second order with respect to our goal and an imprecise and abstracted understanding of the capabilities is totally ok with me. I'll remove that since *you* know it's not necessary.